subprojects/configuration-cache/build.gradle.kts

```diff
@@ -12,11 +12,11 @@ val configurationCacheReportPath by configurations.creating {
 }
 
 dependencies {
-    configurationCacheReportPath(project(":configuration-cache-report"))
+    configurationCacheReportPath(libs.configurationCacheReport)
 }
 
 tasks.processResources {
-    from(configurationCacheReportPath) { into("org/gradle/configurationcache/problems") }
+    from(zipTree(provider { configurationCacheReportPath.files.first() })) { into("org/gradle/configurationcache/problems") }
```

Comment:
Does the published jar also include a manifest? I suppose using `zipTree` would include the manifest into the configuration cache jar, right? So we'd need some exclusions here I think.

Reply:
Right. Now I exclude them.